app/presenters/vendor_api/base.rb

```diff
@@ -0,0 +1,45 @@
+module VendorAPI
+  class Base
+    include Rails.application.routes.url_helpers
+    include VersioningHelpers
+
+    VERSIONS = {}.freeze
+
+    attr_reader :active_version, :available_in_active_version
+
+    def initialize(active_version)
+      @active_version = active_version
+
+      VendorAPI::VERSIONS.each_pair do |version, changes|
+        next unless active_version_in_retrieved_version?(version)
```

Comment:
should we break the loop if the version goes over rather than keep going to the next one?